akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterStressSpec.scala

```diff
@@ -80,7 +80,7 @@ class InterpreterStressSpec extends AkkaSpec with GraphInterpreterSpecKit {
 
     }
 
-    "work with a massive chain of drops" in new OneBoundedSetup[Int](Vector.fill(chainLength / 1000)(Drop(1))) {
+    "work with a massive chain of drops" in new OneBoundedSetup[Int](Vector.fill(chainLength / 1000)(Drop(1)): _*) {
```

Comment:
Yes, this is a difference from PushPullStage (which required an explicit external factory)